File: week00/day02/frequency.py
arr = [4, 2, 4, 3, 2, 4]
freq = {}

for x in arr:
    freq[x] = freq.get(x, 0) + 1

for value, count in freq.items():
    print(value, count)

from collections import Counter
frq = Counter(arr)

File: study_schedule/week00/day02/frequency.cpp
#include <iostream>
#include <vector>
#include <unordered_map>
using namespace std;

int main() {
    vector<int> arr = {4, 2, 4, 3, 2, 4};
    unordered_map<int, int> freq;

    for (int x : arr) {
        freq[x]++;
    }

    for (const auto& [value, count] : freq) {
        cout << value << ": " << count << '\n';
    }

    return 0;
}